tests/classes/core/PKPComponentRouterTest.php

```diff
@@ -260,6 +260,8 @@ public function testRoute()
         // Simulate context DAOs
         $this->_setUpMockDAOs('context1', 'context2');
 
+        $this->expectOutputRegex('/"status":false/');
```

Comment:
Is this an intentional change? I wouldn't expect this PR to affect the router code.

Reply:
This unit test was failing.

Reply:
Can you write a little more about that? I'd worry that the PR added an unexpected behaviour, in which case rather than changing the tests, it would be better to check the behaviour.

Reply:
While running the following command:
```
export XDEBUG_MODE=coverage && php lib/pkp/lib/vendor/phpunit/phpunit/phpunit --configuration lib/pkp/tests/phpunit-env1.xml  --filter testRoute --coverage-html "lib/pkp/tests/results/coverage-html" --debug --verbose
```

Without this `$this->expectOutputRegex('/"status":false/');` lead us to an error (attached below):
![Screenshot from 2021-12-18 20-29-21](https://user-images.githubusercontent.com/1243634/146658196-22bd96c1-1730-4494-a013-fa7b0a8aa4a9.png)

With it, the test passes.
![Screenshot from 2021-12-18 20-31-15](https://user-images.githubusercontent.com/1243634/146658209-685a7976-b417-4577-8035-0503688d4889.png)

Reply:
That suggests to me that your changes are introducing output where there was none before. Have you looked into where that's happening?

Reply:
I'm tracking down. Until Monday I will got more info.